### PR TITLE
Add TX history menu

### PR DESCRIPTION
**Pull request by hsurf22**
### Feature

- [x] Updated CHANGELOG.md
- [x] Is this a user submitted feature request?
  - [x] Link to issue: https://github.com/MyEtherWallet/MyEtherWallet/issues/729
- [x] Add PR label
  - Add TX history menu

### CODE REVIEW

File: src/containers/HeaderContainer/HeaderContainer.vue [hunk 1]
```diff
@@ -249,11 +268,14 @@ export default {
...
-      showGettingStarted: ''
+      showGettingStarted: '',
+      address: '',
+      blockExplorerURL: ''
     };
   },
   computed: {
     ...mapGetters({
+      network: 'network',
       wallet: 'wallet',
```
Comment (kvhnuke) [inline src/containers/HeaderContainer/HeaderContainer.vue]: lets take out all the unused getters, wallet, online..etc

Comment (hsurf22) [reply src/containers/HeaderContainer/HeaderContainer.vue]: Remove 'online' from mapGetters.

File: src/containers/HeaderContainer/HeaderContainer.vue [hunk 2]
```diff
@@ -249,13 +268,15 @@ export default {
       gasPrice: '0',
       error: {},
       resolver: () => {},
-      showGettingStarted: ''
+      showGettingStarted: '',
+      address: '',
```
Comment (gamalielhere) [inline src/containers/HeaderContainer/HeaderContainer.vue]: Not needed as this is set in account global object.

File: src/containers/HeaderContainer/HeaderContainer.vue [hunk 3]
```diff
@@ -249,13 +268,15 @@ export default {
...
+      blockExplorerURL: ''
```
Comment (gamalielhere) [inline src/containers/HeaderContainer/HeaderContainer.vue]: Get this from the network global object

File: src/containers/HeaderContainer/HeaderContainer.vue (hunk 2 above)
Comment (hsurf22) [reply src/containers/HeaderContainer/HeaderContainer.vue]: Removed

File: src/containers/HeaderContainer/HeaderContainer.vue (hunk 3 above)
Comment (hsurf22) [reply src/containers/HeaderContainer/HeaderContainer.vue]: Removed

File: src/containers/HeaderContainer/HeaderContainer.vue [hunk 4]
```diff
@@ -89,6 +89,24 @@
...
+                  v-if="wallet !== null"
+                  right
+                  no-caret
+                  class="tx-history-menu"
+                >
+                  <template slot="button-content">
+                    <p>Transaction History</p>
+                  </template>
+                  <b-dropdown-item :href="blockExplorerURL" target="_blank">
+                    <p>Etherscan (ETH)</p></b-dropdown-item
```
Comment (kvhnuke) [inline src/containers/HeaderContainer/HeaderContainer.vue]: it is not always Etherscan, we have to switch based on the service provider

File: src/containers/HeaderContainer/HeaderContainer.vue [hunk 5]
```diff
@@ -89,6 +89,24 @@
...
+                  >
+                  <b-dropdown-item
+                    :href="'https://ethplorer.io/address/' + address"
+                    target="_blank"
+                    >Ethplorer (Tokens)</b-dropdown-item
```
Comment (kvhnuke) [inline src/containers/HeaderContainer/HeaderContainer.vue]: Ethplorer is only available on `ETH` hide if not ETH

File: src/containers/HeaderContainer/HeaderContainer.vue [hunk 6]
```diff
@@ -248,14 +267,13 @@ export default {
...
       error: {},
-      resolver: () => {},
-      showGettingStarted: ''
+      resolver: () => {}
     };
   },
   computed: {
     ...mapGetters({
+      network: 'network',
       wallet: 'wallet',
```
Comment (kvhnuke) [inline src/containers/HeaderContainer/HeaderContainer.vue]: dont use `wallet` use `account.address` to check whether there is an account

File: src/containers/HeaderContainer/HeaderContainer.vue (hunk 5 above)
Comment (gamalielhere) [reply src/containers/HeaderContainer/HeaderContainer.vue]: Pull `devop/prep-release` so you can use `network.currencyName`

File: src/containers/HeaderContainer/HeaderContainer.vue (hunk 4 above)
Comment (gamalielhere) [reply src/containers/HeaderContainer/HeaderContainer.vue]: You can probably move the `getService` function on SuccessModal to helpers so you can use it here too.

**Pull request merged**